cli2/cli.ts

```diff
@@ -450,20 +465,20 @@ function copyRedistributables(options: CommandOptions, outputPath: string) {
 }
 
 function copyRecursiveSync(src: string, dest: string) {
-  const exists = fs.existsSync(src); 
+  const exists = fs.existsSync(src);
   if (exists) {
-    const stats = exists && fs.statSync(src); 
+    const stats = exists && fs.statSync(src);
     const isDirectory = exists && stats.isDirectory();
     if (isDirectory) {
-      if (!fs.existsSync(dest)) fs.mkdirSync(dest); 
-      fs.readdirSync(src).forEach(function (childItemName) { 
+      if (!fs.existsSync(dest)) fs.mkdirSync(dest);
```

Comment:
Remove exists.

Reply:
fs.exists is deprecated but fs.existsSync.